css/README.md

```diff
@@ -219,7 +220,9 @@ Again: **never nest ID selectors!**
 
 If you must use an ID selector in the first place (and you should really try not to), they should never be nested. If you find yourself doing this, you need to revisit your markup, or figure out why such strong specificity is needed. If you are writing well formed HTML and CSS, you should **never** need to do this.
 
-### Naming conventions
+### Naming conventions (DEPRECATED)
+
+__DEPRECATED__: We keep this section as information about how class names in legacy scss code were structure. Use [CSS Modules](#css-modules) instead.
```

Comment:
structure**d**. i think we could just remove this section altogether, no one needs to write old code anymore so... why keep it?

Reply:
removed it